From 82d884b0cdccb39795b842a39bcc63a05e3ff3d6 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Fri, 31 Jul 2015 08:06:26 +0200
Subject: [PATCH 27/46] Delete pending call watcher

---
 src/accountsservice/accountsmanager.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/accountsservice/accountsmanager.cpp b/src/accountsservice/accountsmanager.cpp
index 2424b3b..17a8851 100644
--- a/src/accountsservice/accountsmanager.cpp
+++ b/src/accountsservice/accountsmanager.cpp
@@ -111,6 +111,7 @@ void AccountsManager::cacheUser(const QString &userName)
     QDBusPendingCallWatcher *watcher = new QDBusPendingCallWatcher(call, this);
     connect(watcher, &QDBusPendingCallWatcher::finished, this, [=](QDBusPendingCallWatcher *w) {
         QDBusPendingReply<QDBusObjectPath> reply = *w;
+        w->deleteLater();
         if (reply.isError()) {
             QDBusError error = reply.error();
             qWarning("Couldn't cache user %s: %s",
-- 
1.9.5

