From f355805ac3c8a2ba21da96a3c6cb7f46579d9072 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Tue, 8 Sep 2015 19:43:02 +0200
Subject: [PATCH 33/46] Specify the language with Travis CI

---
 .travis.yml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.travis.yml b/.travis.yml
index 9f23289..c0e4c4d 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,5 @@
+language: cpp
+
 env:
   global:
    # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
-- 
1.9.5

