From 890afb51a5c77918e4b7440f4c54b293d27c3d4a Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 9 Sep 2015 07:37:30 +0200
Subject: [PATCH 37/46] Silence CMake warnings

---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index aa11e3d..9f1ab3d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,11 @@ project(QtAccountsServices)
 
 cmake_minimum_required(VERSION 2.8.12)
 
+# Silence CMake warnings
+if(POLICY CMP0063)
+    cmake_policy(SET CMP0063 NEW)
+endif()
+
 # Set version
 set(PROJECT_VERSION "0.6.0")
 set(PROJECT_VERSION_MAJOR 0)
-- 
1.9.5

