From 98877c04458b2f1c88d1c58369877e4c999417b4 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 9 Sep 2015 07:38:58 +0200
Subject: [PATCH 38/46] Fix warnings

---
 src/accountsservice/usersmodel.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/accountsservice/usersmodel.cpp b/src/accountsservice/usersmodel.cpp
index 22be3c1..ab3edba 100644
--- a/src/accountsservice/usersmodel.cpp
+++ b/src/accountsservice/usersmodel.cpp
@@ -106,8 +106,6 @@ int UsersModel::rowCount(const QModelIndex &parent) const
 
 QVariant UsersModel::data(const QModelIndex &index, int role) const
 {
-    Q_D(const UsersModel);
-
     UserAccount *user = userAccount(index);
     if (!user)
         return QVariant();
@@ -136,8 +134,6 @@ QVariant UsersModel::data(const QModelIndex &index, int role) const
 
 bool UsersModel::setData(const QModelIndex &index, const QVariant &value, int role)
 {
-    Q_D(UsersModel);
-
     UserAccount *user = userAccount(index);
     if (!user)
         return false;
-- 
1.9.5

