From 25a35cfca6f091f2ca9b43ec38b39ebdd534426d Mon Sep 17 00:00:00 2001
From: Leslie Zhai <xiangzhai83@gmail.com>
Date: Mon, 31 Aug 2015 10:09:06 +0800
Subject: [PATCH 41/46] Fix iconFileName does not exists issue

---
 src/accountsservice/useraccount.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/accountsservice/useraccount.cpp b/src/accountsservice/useraccount.cpp
index 2b3a5d4..bf2fcc1 100644
--- a/src/accountsservice/useraccount.cpp
+++ b/src/accountsservice/useraccount.cpp
@@ -377,7 +377,7 @@ void UserAccount::setShell(const QString &shell)
 QString UserAccount::iconFileName() const
 {
     Q_D(const UserAccount);
-    return d->user->iconFile();
+    return QFile::exists(d->user->iconFile()) ? d->user->iconFile() : QString();
 }
 
 /*!
-- 
1.9.5

