From 1875fee85b369b920170d7111f54985ba58b9f05 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 9 Sep 2015 08:45:34 +0200
Subject: [PATCH 44/46] Reserve lists

Issue: #14
---
 src/accountsservice/accountsmanager.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/accountsservice/accountsmanager.cpp b/src/accountsservice/accountsmanager.cpp
index d7f58b6..4446eaa 100644
--- a/src/accountsservice/accountsmanager.cpp
+++ b/src/accountsservice/accountsmanager.cpp
@@ -193,6 +193,8 @@ UserAccountList AccountsManager::listCachedUsers()
     }
 
     QList<QDBusObjectPath> value = reply.argumentAt<0>();
+    list.reserve(value.size());
+
     for (int i = 0; i < value.size(); i++) {
         const QString path = value.at(i).path();
         UserAccount *account = d->usersCache.value(path, Q_NULLPTR);
@@ -225,7 +227,9 @@ void AccountsManager::listCachedUsersAsync()
                      error.errorString(error.type()).toUtf8().constData());
         } else {
             UserAccountList userList;
+
             QList<QDBusObjectPath> value = reply.argumentAt<0>();
+            userList.reserve(value.size());
             for (int i = 0; i < value.size(); i++) {
                 const QString path = value.at(i).path();
                 UserAccount *account = d->usersCache.value(path, Q_NULLPTR);
-- 
1.9.5

