From 1a5fd91513f045591871fe42728a9ce2e4f0ad0e Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 9 Sep 2015 08:47:05 +0200
Subject: [PATCH 45/46] Do not create QString from const char *

Issue: #14
---
 tests/auto/fakeaccounts.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/auto/fakeaccounts.cpp b/tests/auto/fakeaccounts.cpp
index 9ea01b2..4aa92f2 100644
--- a/tests/auto/fakeaccounts.cpp
+++ b/tests/auto/fakeaccounts.cpp
@@ -90,7 +90,7 @@ QDBusObjectPath FakeAccounts::CreateUser(const QString &name,
                                          const QString &fullName,
                                          int accountType)
 {
-    const QString path = QString("/org/freedesktop/Accounts/User%1").arg(m_lastUid);
+    const QString path = QStringLiteral("/org/freedesktop/Accounts/User%1").arg(m_lastUid);
     FakeUser *entry = new FakeUser(path, m_lastUid, name, fullName, accountType, this);
     new FakeUserAdaptor(entry);
     m_users.append(entry);
-- 
1.9.5

