From 625848f48d7499d2dafdab7fd3541e28371e143d Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 9 Sep 2015 08:48:32 +0200
Subject: [PATCH 46/46] Disable QString casting from const char *

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9f1ab3d..66b2b8b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,9 @@ if(CMAKE_BUILD_TYPE MATCHES "^[Rr]elease$")
     add_definitions(-DQT_NO_DEBUG_OUTPUT)
 endif()
 
+# Definitions
+add_definitions(-DQT_NO_CAST_FROM_ASCII)
+
 # Macros
 include(FeatureSummary)
 
-- 
1.9.5

